quinn-proto/src/connection/mod.rs

```diff
@@ -719,6 +721,13 @@ where
         pad
     }
 
+    /// Returns the source IP used in outgoing packets.
+    ///
+    /// Returns `None` if no specific IP should be used
+    fn src_ip(&self) -> Option<IpAddr> {
```

Comment:
With the config gone, this is now not too useful anymore. Just let me know whether to keep or remove it. I think keeping it is also fine

Reply:
Let's ditch it.

Reply:
removed the `src_ip` functions.